Packet++/header/WireGuardLayer.h

```diff
@@ -0,0 +1,254 @@
+#pragma once
+
+#include "Layer.h"
+#include "IpAddress.h"
+#include "MacAddress.h"
+#include <sstream>
+#include <cstring>
```

Comment:
Are those 2 includes actually used directly in the `WireGuardLayer` header?

If they aren't it would be better to remove them to avoid including them in every translation unit that includes this header.